apps/etterlatte-hendelser-pdl/src/main/kotlin/hendelserpdl/Dodsmeldinger.kt

```diff
@@ -64,12 +63,12 @@ class DodsmeldingerRapid(private val context: RapidsConnection) : IDodsmeldinger
     val logger = LoggerFactory.getLogger(this.javaClass)
 
     override fun personErDod(ident: String, doedsdato: String?, endringstype: Endringstype) {
-        logger.info("Poster at person $ident er doed")
+        logger.info("Poster at en person er doed")
```

Comment:
Hva med å lage en `maskerFnr`-metode som feks viser `ddMMYYYY*****` i steden for å fjerne _helt_? Litt enklere å feilsøke i fremtiden

Reply:
Høres ut som en bedre tilnærming, endrer til det i stedet

Reply:
Oppdatert med en maskerFnr-metode nå!

Reply:
Det er ikke mulig å sette fnr som Foedselsnummer og bruke toString() metoden som gjør det samme som maskerFnr gjør?